liquibase-core/src/main/java/liquibase/database/core/DerbyDatabase.java

```diff
@@ -29,8 +29,8 @@ public DerbyDatabase() {
+        //add reserved words from https://db.apache.org/derby/docs/10.2/ref/rrefkeywords29722.html
```

Comment:
@nvoxland, hi - The list of reserved words contains many fewer words than before. It seems this PR is using the correct list because the list comes from Derby's documentation. Do you agree this is a correct change to make?

Reply:
That is correct. The old list was a much more generic, non-derby specific list. The derby docs should be more correct.